deploy/sdn-ovn-migration/100-network-live-migration.PrometheusRule.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-network-live-migration
+    role: alert-rules
+  name: sre-network-live-migration
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-network-live-migration
+    rules:
+      - expr:
+            (
+              max(
+                max_over_time(
+                  timestamp(
+                    openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+                  )[5d:]
+                )
+              )
+              -
+              min(
+                min_over_time(
+                  timestamp(
+                    openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+                  )[5d:]
+                )
+              )
+            )
+            and on()
+            openshift_network_operator_live_migration_condition{type="NetworkTypeMigrationInProgress"} == 1
+        record: cluster:usage:network_live_migration_duration
+      - expr: openshift_network_operator_live_migration_blocked
+        record: cluster:usage:network_live_migration_blocked
+      - expr: openshift_network_operator_live_migration_condition
+        record: cluster:usage:network_live_migration_condition
+  - name: sre-network-live-migration-alerts
+    rules:
+      - alert: NetworkMigrationDelayedSRE
+        expr: (cluster:usage:network_live_migration_duration > (sum(cluster:node_roles) * 1800))
```

Comment:
@abyrne55 @dakotalongRH should I expect to see `cluster:node_roles` in Telemeter? I am working on the SLO dashboard and I don't seem to be able to find that metric which makes me wonder if this expr works?